src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-91200.mdx

```diff
@@ -0,0 +1,46 @@
+---
+subject: 'Python agent'
+releaseDate: '2024-07-11'
+version: 9.12.0
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: ['Add instrumentation for aiomcache', 'Add support for collect_ai serverside config']
+bugs: ['Fix crash in package capture in 2.7', 'Fix attribute check in OpenAI', 'Fix casing on langchain metric and span names']
+security: []
+---
+
+## Notes
+
+This release of the Python agent adds instrumentation for [aiomcache](https://pypi.org/project/aiomcache/), support for an internally set collect_ai serverside configuration setting, fixes a crash in package catpuring in Python 2.7, an attribute check in [OpenAI](https://pypi.org/project/openai) instrumentation, and casing on langchain metric and span names.
+
+Install the agent using `easy_install/pip/distribute` via the [Python Package Index](https://pypi.python.org/pypi/newrelic) or download it directly from the [New Relic download site](https://download.newrelic.com/python_agent/release).
+
+## New features
+
+* **Add instrumentation for aiomcache**
+
+    * Add instrumentation support for [aiomcache](https://pypi.org/project/aiomcache/). Thank you Jair Henrique for the contribution!
+
+* **Add support for collect_ai serverside config**
+
+    * Add support for an internally set `collect_ai` serverside configuration setting. Note this is returned in the connect response when a particular feature flag is set on an account. 
+
+## Bug fixes
+
+* **Fix crash in package capture in Python 2.7**
+
+    * Previously, when capturing packages on Python 2.7 an UnboundLocalError would be raised. This has been fixed.
+
+* **Fix attribute check in OpenAI**
+
+    * Fix a typo in an attribute check inside the [OpenAI](https://pypi.org/project/openai) instrumentation. Thank you Liam Niehus-Staab for the contribution!
+
+* **Fix casing on langchain metric and span names**
+
+    * Previously, if langchain was being used in combination with an unsupported Large Langage Model library (aka not Bedrock or OpenAI) then the AI dashboard tab would not appear in the UI because the metric name was not cased correctly. This has been fixed and the Span names have been updated to match the casing of the metric name.
```

Comment:
```suggestion
    * Previously, if [LangChain](https://pypi.org/project/langchain/) was being used in combination with an unsupported Large Langage Model library (aka not Bedrock or OpenAI) then the AI dashboard tab would not appear in the UI because the metric name was not cased correctly. This has been fixed and the Span names have been updated to match the casing of the metric name.
```